lib/checker_jobs/notifiers/logger.rb

```diff
@@ -0,0 +1,53 @@
+require 'logger'
+
+class CheckerJobs::Notifiers::Logger
+  DEFAULT = {
+    logdev: STDOUT,
+    level: Logger::INFO
```

Comment:
Style/TrailingCommaInLiteral: Put a comma after the last item of a multiline hash.